pyvista/core/utilities/fileio.py

```diff
@@ -993,6 +997,140 @@ def from_meshio(mesh: meshio.Mesh) -> UnstructuredGrid:
     return grid
 
 
+def to_meshio(mesh: DataSet) -> meshio.Mesh:
+    """Convert a PyVista mesh to a ``meshio`` mesh instance.
+
+    Parameters
+    ----------
+    mesh : pyvista.DataSet
+        Any PyVista mesh/spatial data type.
+
+    Returns
+    -------
+    meshio.Mesh
+        A mesh instance from the ``meshio`` library.
+
+    Raises
+    ------
+    ImportError
+        If the meshio package is not installed.
+
+    Examples
+    --------
+    Convert a pyvista sphere to a ``meshio`` mesh instance.
+
+    >>> import pyvista as pv
+    >>> sphere = pv.Sphere()
+    >>> mesh = pv.to_meshio(sphere)
+
+    """
+    try:
+        import meshio
+
+    except ImportError:  # pragma: no cover
+        raise ImportError('To use this feature install meshio with:\n\npip install meshio')
+
+    try:  # for meshio<5.0 compatibility
+        from meshio.vtk._vtk import vtk_to_meshio_type
+
+    except:  # pragma: no cover
+        from meshio._vtk_common import vtk_to_meshio_type
+
+    # Cast to unstructured grid
+    mesh = mesh.cast_to_unstructured_grid()
+    mesh = (
+        mesh.extract_cells(mesh.cell_data['vtkGhostType'] == 0)
+        if 'vtkGhostType' in mesh.cell_data
+        else mesh
+    )
+
+    vtk_celltypes = mesh.celltypes
+    connectivity = mesh.cell_connectivity
+
+    # Generate polyhedral cell faces if any
+    polyhedral_cells = pyvista.convert_array(mesh.GetFaces())
+
+    if polyhedral_cells is not None:
+        locations = pyvista.convert_array(mesh.GetFaceLocations())
+        polyhedral_cell_faces = []
+
+        for location in locations:
+            if location == -1:
+                continue
+
+            n_faces = polyhedral_cells[location]
+            i = location + 1
+            faces: list[MatrixLike[int]] = []
+
+            while len(faces) < n_faces:
+                n_vertices = polyhedral_cells[i]
+                faces.append(polyhedral_cells[i + 1 : i + 1 + n_vertices])
+                i += n_vertices + 1
+
+            polyhedral_cell_faces.append(faces)
+
+    # Single cell type (except POLYGON and POLYHEDRON)
+    if vtk_celltypes.min() == vtk_celltypes.max() and vtk_celltypes[0] not in {7, 42}:
+        vtk_celltype = vtk_celltypes[0]
+        cells = connectivity.reshape((mesh.n_cells, connectivity.size // mesh.n_cells))
+
+        if vtk_celltype == 8:
+            cells = cells[:, [0, 1, 3, 2]]
+            celltype = 'quad'
```

Comment:
```suggestion
        if vtk_celltype == pv.CellType.PIXEL:
```
Can we use the [CellType](https://dev.pyvista.org/api/utilities/_autosummary/pyvista.celltype#pyvista.CellType) enum here instead?

Reply:
Whoops this suggestion accidentally removed the code block, sorry!